test/smart_city/data_test.exs

```diff
@@ -200,15 +201,12 @@ defmodule SmartCity.DataTest do
         }
       }
 
-      allow(Data.Timing.measure(app, label, any()),
-        exec: fn _a, _l, f ->
-          {:ok, result} = f.()
-          {:ok, result, timing}
-        end,
-        meck_options: [:passthrough]
-      )
-
-      assert Data.timed_new(json_message, app) == {:ok, expected_message}
+      with_mock(Data.Timing, [:passthrough], [measure: fn (app, label, function) ->
+         {:ok, result} = function.()
```

Comment:
These github action warnings are concerning. It only does this as part of the github action, it doesn't run the tests. Have you done that locally? I imagine elixir would complain about unused vars

Reply:
These I kinda skipped over, as they were warnings when I run tests. Didn't realize that they would show here (a part of upgrading, perhaps). Anyway, this was more of a learning thing from the new mocking, but have been taken care of now.